ext/mixed/display-templates.html

```diff
@@ -28,11 +28,7 @@
     <div class="tags tag-list"></div>
     <div class="frequencies tag-list"></div>
 </div></div></template>
-<template id="term-definition-item-template"><li class="term-definition-item">
-    <div class="term-definition-tag-list tag-list"></div>
-    <div class="term-definition-disambiguation-list"></div>
-    <ul class="term-glossary-list"></ul>
-</li></template>
+<template id="term-definition-item-template"><li class="term-definition-item"><div class="term-definition-tag-list tag-list"></div><div class="term-definition-disambiguation-list"></div><ul class="term-glossary-list"></ul></li></template>
```

Comment:
Not needed for this PR

Reply:
I wish; this is honestly one of the most annoying parts about HTML and layouts. A potential alternative is something like this:
```html
<div><!--
    --><div></div><!--
    --><div></div><!--
--></div>
```

but even that is rather undesirable.

Reply:
4c1514bdb545a75c41c299a263e2f13b2d48828d

(We could also use XHTML :^)

Reply:
> I wish; this is honestly one of the most annoying parts about HTML and layouts. A potential alternative is something like this:
> 
> ```html
> <div><!--
>     --><div></div><!--
>     --><div></div><!--
> --></div>
> ```
> 
> but even that is rather undesirable.

Yeah, that's pretty ugly :smile:

Handlebars has the `{{~...~}}` syntax, but for our use case maybe a class could be used for indicating whitespace stripping.

Reply:
Might be a good idea, but also depends on the performance. DOM instantiation is already a point of slowness, particularly since it's not reused. Also might get a bit tricky since there are some places where the spaces are intentional. Worth looking into at some point.

Or if there's some good way to validate HTML as part of CI (i.e. no missing end tags and such), that could also be helpful.